crates/cext/src/exit_codes.rs

```diff
@@ -43,6 +44,20 @@ pub enum ExitCode {
     MismatchedQubits = 201,
     /// Matrix is not unitary.
     ExpectedUnitary = 202,
+    /// Target related error
+    TargetError = 300,
+    /// Instruction already exists in the Target
+    TargetInstAlreadyExists = 301,
+    /// Properties with incorrect qargs was added
+    TargetQargMismatch = 302,
+    /// Trying to query into the target with non-existent qargs.
+    TargetInvalidQargsKey = 303,
+    /// Querying an operation that doesn't exist in the Target.
+    TargetInvalidInstKey = 304,
+    /// An attempt to add a parametric gate without fixed parameters.
+    TargetNonFixedParametricGate = 305,
+    /// An attempt to add a non-parametric gate with parametric methods
+    TargetNonParametricError = 306,
```

Comment:
Are these 2 actually used anywhere?

Reply:
I think not anymore, I will remove them.